### PR TITLE
Implement MusicAlbum and Genre Class

### PR DESCRIPTION
**Pull request by bienvenuushindi**
- #19 
- #20 
- #21 
- #23

### CODE REVIEW

**Review by ChrispaixK — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @bienvenuushindi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/genre.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    @id = 0
+    @name = name
+    @items = items
+  end
+
+  def add_item(item)
+    @items << item
+    item.add_genre = self
+  end
+end
```
Comment (pinkmoon25) [inline classes/genre.rb:15]: [OPTIONAL] I just wanted to ask where are we pushing the items to? isn't `@items` is supposed to be an array.
basically where are we adding the items with the add_item method. 
Thanks

**Review by pinkmoon25 — APPROVED** [review]
## STATUS: APPROVED :green_circle: 
### Hi @bienvenuushindi wave,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: 

### To Highlight 
No linter errors. :heavy_check_mark: 
Gitflow is followed. :heavy_check_mark: 
Professional README file. :heavy_check_mark: 
PR has a good title and summary. :heavy_check_mark: 


Cheers and Happy coding! :clap: :clap: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

### Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application.

File: classes/genre.rb (hunk 1 above)
Comment (bienvenuushindi) [reply classes/genre.rb:15]: @PinkMoon25 Thank you for your question
You are right `@item` is an array reason why you can see the use of ` @item << item `in this function `add_item`
If you look well at the constructor method I assumed that sometimes you may want to create a genre instance with some items inside You can do that by passing an array of items as the constructor's second argument.
For instance, let's have look at this example which illustrates what I am saying
`Genre.new("name",[item,item])`

**Pull request merged**